testsuite/tests/kuadrant/authorino/operator/conftest.py

```diff
@@ -49,4 +49,5 @@ def authorino(openshift, blame, request, testconfig, label, authorino_parameters
     request.addfinalizer(authorino.delete)
     authorino.commit()
     authorino.wait_for_ready()
+    authorino.refresh()
```

Comment:
Optionally, you can add it to wait_for_ready in authorino directly, or add it to wait_until so every wait is refreshed

Reply:
Yes this seems like better idea. So its called just in one place.